site/installing.md

```diff
@@ -261,17 +187,11 @@ Next, create a dedicated project for Weave Scope then apply policy changes neede
 The installation method for Scope on OpenShift is very similar to the one described above for Kubernetes, but instead of `kubectl apply ...` you need to use
 `oc apply ...` and install it into the namespace of the `weave-scope` project you have just created, and not the `weave` namespace, i.e.:
 
-If you are to use Weave Cloud run this command:
-
-    oc apply -f 'https://cloud.weave.works/k8s/scope.yaml?service-token=<token>'
-
-And if you are to deploy Scope in standalone mode run:
-
     oc apply -f 'https://cloud.weave.works/k8s/scope.yaml'
 
-To access standalone Scope app from the browser, please refer to Kubernetes instructions above.
+To access the Scope app from the browser, please refer to Kubernetes instructions above.
 
```

Comment:
Should link to the Kubernetes instructions that is being referred to.

Reply:
Fixing.